core/templates/domain/admin/admin-backend-api.service.ts

```diff
@@ -495,7 +495,10 @@ export class AdminBackendApiService {
   }
 
   async revokeSuperAdminPrivilegesAsync(username: string): Promise<void> {
-    return this.http['delete']<void>(
+    // This gives ".delete is a syntax error" due to eslint.
+    // For more information visit https://eslint.org/docs/rules/dot-notation
+    // eslint-disable-next-line dot-notation
+    return this.http.delete<void>(
```

Comment:
Can we set `allowedKeywords` to true? ([ref](https://github.com/eslint/eslint/issues/7457#issuecomment-256514376))

Reply:
If required we can do it in a separate PR! (I would prefer doing that) and let's avoid having multiple unexpected changes in this PR!

Reply:
done, created a separate for this